src/screens/dialogs/access-file-dialog.tsx

```diff
@@ -0,0 +1,80 @@
+import { useContext, useState, useEffect } from "react";
+import { SharedWithWhoService } from "../../services/files/get-users-file.service";
+import { Dialog } from "../../components/Dialog";
+import {
+  FilesDialogsContext,
+  AVAILABLE_DIALOGS,
+  AuthContext
+} from "../../context/index";
+
+export const AccessManagementDialog = () => {
+  const [usersWithAccess, setUsersWithAccess] = useState<string[]>([]);
+
+  const { dialogsVisibilityState, closeDialog, selectedFile } =
+    useContext(FilesDialogsContext);
+  const Open = dialogsVisibilityState[AVAILABLE_DIALOGS.ACCESS_MANAGEMENT];
+
+  const [_newAccess, setNewAccess] = useState(selectedFile?.uuid || "");
```

Comment:
The `selectedFile` is an state by itself. Avoid creating a [derivative state](https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state).